wavefront/server/background_jobs/celery_worker/celery_worker/tasks/trigger_event_task.py

```
@@ -0,0 +1,46 @@
+    asyncio.set_event_loop(loop)
+    try:
+        return loop.run_until_complete(
+            processor.process(trigger_id=UUID(trigger_id), raw_payload=raw_payload)
+        )
+    except Exception as exc:
+        logger.exception(
+            f'process_trigger_event_task failed for trigger {trigger_id} '
+            f'(push_message_id={push_message_id}): {exc}'
+        )
+        if self.request.retries < self.max_retries:
+            raise self.retry(exc=exc)
```

Ruff (0.15.13)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)